packages/web-service/src/pages/habitat/a24/sett/sett-active-entrance-total.js

```diff
@@ -9,6 +9,7 @@ import { checkApplication } from '../../../common/check-application.js'
 import { isCompleteOrConfirmed } from '../../../common/tag-functions.js'
 import { A24_SETT } from '../../../tasklist/a24-badger-licence.js'
 
+const oldKey = 'habitat-active-entrances'
```

Comment:
This is needed in some places to allow the existing code to function. 

This is coupled to the fields in the html, validator and session key but has not been configured to be used everywhere and is hard coded in multiple places (error summary / html etc).

It might be good to inject this into the page for the id's and use on the session key to create that unique link with the page (doing this might help us 'lift' these pages into a more common template / view. 

I think keep this key as it's previous value for now makes sense as i think it affects the json column stores in the db. But i think changing the 'magic-string' would still be beneficial.

Reply:
Applies to all `oldKey` occurrences